spec/inputs/beats_spec.rb

```diff
@@ -166,7 +166,14 @@
   end
 
   context "tls meta-data" do
-    let(:config) { super().merge("host" => host, "ssl_peer_metadata" => true, "ssl_certificate_authorities" => [ certificate.ssl_cert ]) }
+    let(:config) do
+      super().merge(
+          "host" => host,
+          "ssl_peer_metadata" => true,
+          "ssl_certificate_authorities" => [ certificate.ssl_cert ],
+          "ecs_compatibility" => 'disabled'
```

Comment:
we don't have a test case for tls metadata in ecs mode, isn't it?

Reply:
might be elsewhere - recall writing this spec as specifically guarding against a [NPE regression](https://github.com/logstash-plugins/logstash-input-beats/pull/431), 
the bug surfaced regardless of `ecs_compatibility` and the test here isn't about verifying ECS vs non-ECS field behavior.